content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -257,6 +257,20 @@ Your applications cannot expect any performance SLAs (disk IOPS for example)
 from local ephemeral storage.
 {{< /caution >}}
 
+{{< note >}}
+This is an exception for enforcing resource quota on ephemeral-storage.
+
```

Comment:
these two lines can be safely dropped?
It is confusing what is an exception.

Reply:
You can see the preview at here:
https://deploy-preview-35011--kubernetes-io-main-staging.netlify.app/docs/concepts/configuration/manage-resources-containers/